app/components/ui/files/publish-modal/component.js

```diff
@@ -1,326 +1,413 @@
-            hoverable: true,
-            html: "The environment archive holds the information needed to re-create the tale's base virtual machine."
-        });
-        // Create the LICENSE popup
-        $('.info.circle.blue.icon.license\\.txt').popup({
-            position: 'right center',
-            target: '.info.circle.blue.icon.license\\.txt',
-            hoverable: true,
-            html: "Each package is created with a license, which can be selected below."
-        });
-        // Create the science_metadata popup
-        $('.info.circle.blue.icon.science_metadata\\.xml').popup({
-            position: 'right center',
-            target: '.info.circle.blue.icon.science_metadata\\.xml',
-            hoverable: true,
-            html: "The contents of each package are described using the Ecological Metadata Language (EML). \
-             To learn more about EML, visit the \
-             <a href='https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/0012-9623%282005%2986%5B158%3AMTVOED%5D2.0.CO%3B2' \
-              target='_blank'>EML primer</a>."
-        });
+    // A map that connects the repository dropdown to a url
+    self.repositories = [{
+      name: 'DataONE Development',
+      url: 'https://dev.nceas.ucsb.edu/knb/d1/mn'
     },
-
-    joinArray(arr) {
-        let joined = String();
-        arr.forEach(function (item) {
-            joined += "'" + item + "'";
-        });
-        return joined;
+    {
+      name: 'DataONE-The Knowledge Network for Biocomplexity',
+      url: 'https://knb.ecoinformatics.org/knb/d1/mn'
     },
-
-    prepareItemIds() {
-        /* let itemIdList = [];
-        this.get('fileList').forEach(function(item) {
-            itemIdList.push(JSON.stringify(item.id));
-        }); */
-        return this.get('inputData');
-    },
-
-    getRepositoryPathFromName(name) {
-        // Given a repository name, find the membernode URL
-        let repostoryList = this.get('repositories')
-        for (var i = 0; i < repostoryList.length; i++) {
-            if (repostoryList[i].name === name) {
-                return repostoryList[i].url;
+    {
+      name: 'DataONE-Arctic Data Center',
+      url: 'https://arcticdata.io/metacat/d1/mn'
+    }];
+
+    self.get('store').findRecord('tale', self.get('modalContext'), {
+        reload: false
+      })
+      .then(resp => {
+        self.set('tale', resp);
+        // We want to check if the last publish event is still running
+        let lastJob = self.get('internalState').getLastPublishJob();
+        
+        if (lastJob) {
+          self.get('store').findRecord('job', lastJob, {
+            reload: true}).then(jobResp => {
+              if (jobResp && jobResp.status == 2) {
+              // Check if the job is publishing this particular tale
+              self.talePublishing(lastJob);
+            } else if (resp.published) {
+              self.talePublished(resp);
             }
+          }).catch(()=>{
+            
```

Comment:
Empty catch clause here... please either remove this or add a `console.log()` that an error occurred, or similar?